package.json

```diff
@@ -3,11 +3,15 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@material-ui/core": "^4.9.5",
+    "@material-ui/icons": "^4.9.1",
```

Comment:
This dependencies might be deleted. At least, I haven't found any usages in the code